Resources/Prototypes/_Goobstation/Hydroponics/seeds.yml

```
@@ -0,0 +1,63 @@
+    Nutriment:
+      Min: 1
+      Max: 20
+      PotencyDivisor: 20
+    Water:
+      Min: 1
+      Max: 4
+      PotencyDivisor: 25
+    Vitamin:
+      Min: 1
+      Max: 2
+      PotencyDivisor: 50
```

yamllint (1.35.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)